get_deps.sh

```diff
@@ -16,6 +16,7 @@ if [[ $1 == --help || $1 == help ]]; then
 		Argument variables:
 		VERBOSE=1       Print commands
 		FORCE=1         Download even if present
+		WITH_DLPACK=0   Skip dlpack
```

Comment:
I don't think we can get away from DLPACK. It is not relative to the backend, it's an essential component for the RedisAI core module.

Reply:
I know that. This flag is merely for diagnostic purposes, to prevent get_deps.sh from checking it out when FORCE=1 and we're aiming for the script logic, not for the subsequent build.